frontend/src/deviceDetailsPage/__tests__/DeviceDetailsPageContents.test.jsx

```diff
@@ -24,11 +28,23 @@ describe("<DeviceDetailsPageContents/> functional component", () => {
         <DeviceDetailsPageContents device={dummyDevice} />
       );
     });
-    it("Contains 3 Grid Components", () => {
-      expect(wrapper.find(Grid)).toHaveLength(3);
+    it("Contains 4 Grid Components", () => {
+      expect(wrapper.find(Grid)).toHaveLength(4);
```

Comment:
is it worth it to check the grids' props?

Reply:
yes, it is. i'm just being lazy because I wanted to be able to actually start my actual task